### PR TITLE
DOC: add juno3 flight example

### PR DESCRIPTION
**Pull request by Gui-FernandesBR**
## Pull request type

- [x] ReadMe, Docs and GitHub updates

## Checklist

- [ ] Tests for the changes have been added (if needed)
- [x] Docs have been reviewed and added / updated
- [x] Lint (`black rocketpy/ tests/`) has passed locally 
- [x] All tests (`pytest --runslow`) have passed locally
- [ ] `CHANGELOG.md` has been updated (if relevant)

## New behavior
A new flight was added to the Flight Examples section on our documentation.

## Breaking change

- [x] No

## Additional information
image

### CODE REVIEW

File: data/weather/spaceport_america_pressure_levels_2023_hourly.nc
Comment (Gui-FernandesBR) [inline data/weather/spaceport_america_pressure_levels_2023_hourly.nc:1]: Commenting here so I don't forget: this file weights more than 20MB, I still need to compress the .nc file so we can save cloud space.

File: docs/examples/juno3_flight_sim.ipynb
Comment (MateusStano) [inline docs/examples/juno3_flight_sim.ipynb:1]: Regarding the notebook:

- `thrust_source` string path is not generalized for the repo
- Missing a `pandas` import
- The cell that creates the dataframes is breaking

Comment (Gui-FernandesBR) [reply docs/examples/juno3_flight_sim.ipynb:1]: ty for noticing this. That Pandas cell was left behind by mistake, I fixed it now.

File: data/weather/spaceport_america_pressure_levels_2023_hourly.nc
Comment (Gui-FernandesBR) [reply data/weather/spaceport_america_pressure_levels_2023_hourly.nc:1]: No I didn't compress this time, but we can compress later and clean the git history.

http://climate-cms.wikis.unsw.edu.au/NetCDF_Compression_Tools

**Comment by Gui-FernandesBR** [comment]
@MateusStano we need an "approve" to move along and merge this one.

**Review by MateusStano — APPROVED** [review]

**Pull request merged**